lib/matplotlib/textpath.py

```diff
@@ -358,11 +349,21 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
                     warnings.warn("No supported encoding in font (%s)." %
                                   font_bunch.filename)
 
+                # Character is a glyph and needs to be mapped to corresponding index
                 if charmap_name == "ADOBE_STANDARD" and font_bunch.encoding:
+                    use_glyph = True
                     enc0 = dviread.Encoding(font_bunch.encoding)
-                    enc = {i: self._adobe_standard_encoding.get(c, None)
-                           for i, c in enumerate(enc0.encoding)}
+
+                    # Make a list of each glyph by splitting the encoding
+                    enc0_list = []
+                    for e in enc0.encoding:
+                        enc0_list += e.split("/")
+
+                    # Encoding provided by the font file mapping names to index
+                    enc = {i: font.get_name_index(c) or None
```

Comment:
Yes, this is the block where `if charmap_name == "ADOBE_STANDARD" and **font_bunch.encoding**:`
Since `**font_bunch**` already has Unicode values in them, we don't need to specially use the adobe standard file.
Thus, it was removed completely.